### PR TITLE
feat: Align enums (netstandard2.0)

### PR DESCRIPTION
**Pull request by ebariche**

### CODE REVIEW

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UWP/ApplicationModel/Contacts/ContactQueryDesiredFields.cs [hunk 1]
```diff
@@ -6,23 +6,23 @@ namespace Windows.ApplicationModel.Contacts
 	/// Defines which fields must exist on a contact in order to match a search operation.
 	/// </summary>
 	[Flags]
-	public enum ContactQueryDesiredFields
+	public enum ContactQueryDesiredFields : uint
```
Comment (carldebilly) [inline src/Uno.UWP/ApplicationModel/Contacts/ContactQueryDesiredFields.cs:9]: Why are you converting `enum`s to use `uint`?

**Review by carldebilly — COMMENTED** [review]
I'm very cir

File: src/Uno.UWP/ApplicationModel/Contacts/ContactQueryDesiredFields.cs (hunk 1 above)
Comment (ebariche) [reply src/Uno.UWP/ApplicationModel/Contacts/ContactQueryDesiredFields.cs:9]: Because they should be uints :)

Comment (ebariche) [reply src/Uno.UWP/ApplicationModel/Contacts/ContactQueryDesiredFields.cs:9]: image

**Pull request merged**